lib/rest-adapter.js

```diff
@@ -571,16 +577,16 @@ RestAdapter.prototype.allRoutes = function() {
 };
 
 RestAdapter.prototype.getClasses = function() {
-  return this.remotes.classes(this.options).map(function(c) {
-    return new RestClass(c);
+  return this.remotes.classes(this.options).map(c => {
+    return new RestClass(c, this.options);
```

Comment:
Hmm but this now breaks the app-level setting of `normalizeHttpPath` again in my app. : / I really don't understand enough about all the involved classes, use cases and involved complexities to fix this properly. I will try to figure out what's missing now...